src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaData.java

```diff
@@ -640,8 +644,315 @@ private void addImportedKey(final List<List<Object>> metaData,
 
     @Override
     public ResultSet getTypeInfo() throws SQLException {
-        // TODO: Implement
-        throw new SQLFeatureNotSupportedException();
+        int UNUSED = -1;
```

Comment:
Can define UNUSED, BASE_10 and BASE_2 as private static final and declare at top of class